includes/Shortcode/Stories_Shortcode.php

```diff
@@ -61,10 +61,10 @@ public function register(): void {
 	 *
 	 * @since 1.5.0
 	 *
-	 * @param array<string,string|int> $attrs Shortcode attributes.
+	 * @param array<string,string|int>|string $attrs Shortcode attributes.
 	 * @return string Story markup.
 	 */
-	public function render_stories( array $attrs ): string {
+	public function render_stories( mixed $attrs ): string {
```

Comment:
Note that `mixed` is only supported since PHP 8.0, but we still support PHP 7.4.

We can simply omit the type hint here